src/components/NewMessageForm/NewMessageFormPollEditor.vue

```diff
@@ -24,83 +25,90 @@
 	<NcModal size="small"
 		:container="container"
 		v-on="$listeners">
-		<div class="simple-polls-editor">
+		<div class="poll-editor">
 			<h2>{{ t('spreed', 'Create new poll') }}</h2>
 
 			<!-- Poll Question -->
-			<p class="simple-polls-editor__caption">
+			<p class="poll-editor__caption">
 				{{ t('spreed', 'Question') }}
 			</p>
 			<NcTextField :value.sync="pollQuestion" :label="t('spreed', 'Ask a question')" v-on="$listeners" />
 
 			<!-- Poll options -->
-			<p class="simple-polls-editor__caption">
+			<p class="poll-editor__caption">
 				{{ t('spreed', 'Answers') }}
 			</p>
-			<PollOption v-for="(option, index) in pollOptions"
+			<div v-for="(option, index) in pollOptions"
 				:key="index"
-				:ref="`pollOption${index}`"
-				class="simple-polls-editor__option"
-				:value.sync="pollOptions[index]"
-				:label="t('spreed', 'Answer {option}', {option: index + 1})"
-				:can-delete="pollOptions.length > 2"
-				@delete-option="deleteOption(index)" />
+				class="poll-editor__option">
+				<NcTextField :ref="`pollOption${index}`"
```

Comment:
```suggestion
				<NcTextField ref="pollOption"
```

Both Vue 2 and Vue 3 support using ref with `v-for`, the result is an array.

https://v2.vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements

https://vuejs.org/guide/essentials/template-refs.html#refs-inside-v-for

It is a bit more clean (just an array, no creating strings with index inside) and easier to use with Composition API.